webapp/src/containers/createRequest/index.tsx

```diff
@@ -123,8 +90,51 @@ const CreateRequest = () => {
         return {isLoading, isSuccess, isError, data: data as ConfigData | undefined, error: (apiErr as FetchBaseQueryError)?.data as APIError | undefined};
     };
 
-    const serviceNowBaseURL = getConfigState().data?.ServiceNowBaseURL;
+    const getItemsSuggestions = () => {
+        const {isLoading, isSuccess, isError, data, error: apiErr} = getApiState(Constants.pluginApiServiceConfigs.searchItems.apiServiceName, searchItemsPayload);
+        return {isLoading, isSuccess, isError, data: data as RequestData[], error: (apiErr as FetchBaseQueryError)?.data as APIError | undefined};
+    };
+
+    // Get the suggestions from the API
+    const getSuggestions = ({searchFor}: {searchFor?: string}) => {
+        setApiError(null);
+        setRequest(null);
+        setSearchItemsPayload({search: searchFor || ''});
+        makeApiRequest(Constants.pluginApiServiceConfigs.searchItems.apiServiceName, {search: searchFor || ''});
+    };
 
+    const debouncedGetSuggestions = useCallback(Utils.debounce(getSuggestions, 500), [getSuggestions]);
```

Comment:
There should be a constant for `500` and it should be used in all the search fields.